### ISSUE TITLE
Create "Wizard" system at create command.

### ISSUE THREAD

**Issue by diogocezar** [neutronjs/neutron#7]
Would be nice, if some questions could be asked on command `neutron create`

These things could be considered:

- ESLint style options;
- Would you like use saga?
-  Would you like thunk?
- Would you like to use duck system? Ou tradicional files?
- Would you like to use styled-components?
- Use statefull components with: Classes or Hooks?
- Editor config styles;
- Would you like to add jest?
- Custom fav icon and index.html template?
- Would you like to configure PWA with service worker?
- Maybe some CSS architecture like RSCCS or BEM?
- Would you like to configure some deploy system? GitHub Pages? S3 Aws?

This options also could be skipped with some argument like `--defaults`

**Comment by thrsouza** [comment]
@diogocezar, yeah! I think would be nice too. 

We'll evaluate and study all items to implement them. 

Thanks and let's do it! 🚀

**Comment by papuruth** [comment]
I agree on this please consider adding Service Worker in next update
